src/net/sourceforge/kolmafia/persistence/DailyLimitDatabase.java

```diff
@@ -112,7 +122,7 @@ public int getMax() {
         }
       }
 
-      return 1;
+      return -1;
```

Comment:
This looks like a magic number change to me, having not looked at every file yet.  Given my expectations for something named getMax I would expect it always returned a non-negative integer.  If it is used in the context of "can I do this" then I would expect zero to mean No.  So I'd like to see this return a zero (if things fall through to here) or make a named constant so it is clear that -1 has a special meaning.

Reply:
> Given my expectations for something named getMax I would expect it always returned a non-negative integer.

Your expectations are correct - this is what is returned if we have invalid underlying data. I think it's fine to keep it as is.

Reply:
I would prefer, then, a named constant or a comment that said something like "-1 is understood elsewhere to mean that there was invalid underlying data".

I also wonder, because I have not spent a lot of time on context for this, if "invalid underlying data" should be logged as an error so the underlying data could be corrected?

Reply:
It is logged, it's just done in the calling function which is where the data is actually being loaded. I could always make it throw or something but this seems fine

Reply:
I'll add a comment